packages/compiler/vitest.config.ts

```diff
@@ -1,6 +1,8 @@
+import tsconfigPaths from 'vite-tsconfig-paths'
 import { defineConfig } from 'vitest/config'
 
 export default defineConfig({
+  plugins: [tsconfigPaths()],
```

Comment:
we might not need rollup. Don't remove yet tho

Reply:
The tests do not configure the alias without this, so it does not find the imported files and it fails.

Reply:
ahhh sorry this is vitest. ignore